app/forms/candidate_interface/degree_wizard.rb

```diff
@@ -52,6 +55,26 @@ class InvalidStepError < StandardError; end
     validate :award_year_in_past_when_degree_incomplete, on: :award_year
     validate :award_year_after_teacher_training_starts, on: :award_year
 
+    def country
+      COUNTRIES_AND_TERRITORIES.key(@country_raw || @country) || @country
+    end
+
+    def subject
+      @subject_raw || @subject
+    end
+
+    def other_type
+      @other_type_raw || @other_type
```

Comment:
Do the _raw attributes need to be instance variables? if they've got attr_readers

Reply:
Good question. :) I always choose to use instance var because they are quicker than calling the reader (one less method call). But I don't mind either way. Do you want me to change to call `other_type_raw`?

Reply:
happy to for them to be left as instance variables, i didn't know they had better performance 👍

Reply:
happy to for them to be left as instance variables, i didn't know they had better performance 👍